spec/src/main/asciidoc/configexamples.asciidoc

```diff
@@ -121,6 +121,11 @@ public class InjectedConfigUsageSample {
     @ConfigProperty(name="myprj.some.supplier.timeout", defaultValue="100")
     private java.util.function.Supplier<Long> timeout;
 
+    //Injects a Map to resolve multiple configuration parameters under the same configuration key.
+    @Inject
+    @ConfigProperty(name = "myprj.some.reasons", defaultValue = "200=OK;201=Created")
+    private java.util.Map<Integer, String> reasons;
```

Comment:
´´´suggestion
private java.util.Map<String, String> reasons;
´´´
Once we're talking about configuration I believe the key must be a `String`

Reply:
I tend to agree to narrow down to Map<String, String>. Otherwise, it will be overwhelming with the tests and it might have more issues to deal with regarding conversion errors.